doc/reference.conf

```
@@ -546,12 +546,17 @@ operator "god" {
 	privset = "admin";
 };
 
-/* connect {}: controls servers we connect to (OLD C:, N:, H:, L:) */
+/* connect {}: controls servers we connect with (OLD C:, N:, H:, L:).
+ *
+ * This configuration is used whether connections are incoming or
+ * outgoing. */
 connect "irc.uplink.com" {
 	/* the name must go above */
 
-	/* host: the host or IP to connect to.  If a hostname is used it
-	 * must match the reverse dns of the server.
+	/* host: the host or IP to connect to.
+     *
+     * It is also used to validate incoming connections. If a hostname
+     * is used, it must match the reverse dns of the server.
```

This has mixed indentation

```suggestion
	/* host: the host or IP to connect to.
	 *
	 * It is also used to validate incoming connections. If a hostname
	 * is used, it must match the reverse dns of the server.
```